### PR TITLE
added some more information to the zenodo metadata file; refs #207

### PR DESCRIPTION
**Pull request by jspaaks**
**Related issues**

Refs: #207, #219

**Describe the changes made in this pull request**

- added more metadata to `.zenodo.json`, e.g. the contributors information, the references, and the related identifiers so we don't have to do this by hand after release. 
- removed cffconvert workflow, because `.zenodo.json` and `CITATION.cff` are now expected to be slightly different

**Instructions to review the pull request**

- look at the file diff
- Testing on sandbox could also work, but then you need to remove the `related_identifiers` section.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+        {
+            "affiliation": "Octiog, Inc., USA",
+            "name": "Crusoe, Michael R.",
+            "orcid": "0000-0002-2961-9670",
+            "type": "Other"
+        }
+    ],
     "creators": [
         {
+            "affiliation": "Humboldt-Universitaet zu Berlin, Department of German Studies and Linguistics, Berlin, Germany",
```
Comment (sdruskat) [inline .zenodo.json:29]:
```suggestion
            "affiliation": "German Aerospace Center (DLR), Berlin, Germany",
```

**Review by sdruskat — APPROVED** [review]
Thanks! Approved after I changed my affiliation :)

**Comment by jspaaks** [comment]
Merged before all tests finished, let;s hope the GH gods are with us

**Comment by sdruskat** [comment]
> Merged before all tests finished, let;s hope the GH gods are with us

Ha, when if not today!

**Pull request merged**